### ISSUE TITLE
update What to expect at a meeting section

### ISSUE THREAD

**Issue by Alexandra-Haynes** [CodeWithAloha/CWAWebsite#28]
As we are no longer part of Code for America (they shut down their brigades system), we need to update the _What to expect at a meeting_ copy. All 1,2 and 3 bullets have to be rewritten to better reflect our organization and set a welcoming environment for new people thinking about joining us. 

**My idea:** 

**1**. At every meeting, we come together from various locations, united by a common goal. We begin by acknowledging the land and communities from where we all connect. This moment of reflection sets a respectful and inclusive tone for our time together. Here, we would love to hear your name, learn where you are calling from, and listen to your response to our question of the day. This interaction enhances our bond and understanding, setting the stage for a productive and enjoyable meeting.

**2**. In our meetings, we keep everyone in the loop about essential project milestones, updates, and challenges. This transparency allows us to work together effectively to overcome any obstacles in our path. But that’s not all – we also provide a platform for guests and partners to share their ongoing projects. This opportunity allows us all to gain insights and expand our knowledge about making a meaningful impact right here in Hawaii. By staying informed and connected, we continuously enhance our contribution to the growth and betterment of our beautiful state.

**3.** During each meetup, we facilitate breakout rooms tailored to our ongoing projects. These dedicated spaces provide the perfect environment for targeted discussion, planning, and problem-solving, propelling us closer to our milestones. Embrace the opportunity to delve deep, collaborate intensively, and contribute significantly to projects that resonate with you. Together, in these breakout rooms, we drive change, foster innovation, and make substantial progress, contributing to the betterment and growth of our community. Your insights and efforts can make a significant difference – let's achieve brilliance together!

![Screenshot 2023-09-24 124024](https://github.com/CodeWithAloha/CWAWebsite/assets/113944962/d2ec4c07-e4c0-454d-9dd6-957bfbd5c32b)

Please comment below if you would like to work on this issue and we will assign you to it 😄

**Comment by AnaBaldinger** [comment]
@Alexandra-Haynes  I'm happy to help with this! I liked what you wrote in the notes above, so I used this and made updates to condense each paragraph so it's easier to read through when scrolling through the page. Lmk what you think: 

1. Meet & Greet: "At every meeting, we come together from various locations, united by a common goal. We begin by acknowledging the communities we come from, ensuring that we set a respectful and inclusive environment. Afterward, each of the group members introduces themselves and we all discuss a question of the day."

2. Project Updates: "In our meetings, we keep everyone updated on project milestones, challenges, and progress. We also give guests and partners a platform to share their ongoing work, allowing us to expand our knowledge and impact. By staying connected, we continuously improve our contribution to Hawaii."

3. Breakout Rooms: "During each meetup, we facilitate breakout rooms tailored to our projects for targeted discussion and planning. Together in these rooms, we drive innovation and progress, contributing to the growth and betterment of our community. Your insights can make a real difference – let's achieve greatness together!"
